.aglintrc.yaml

```diff
@@ -9,6 +9,9 @@ rules:
     - error
     - regexp-patterns:
         # Do not forget to escape special regex characters
+        # https://github.com/AdguardTeam/AdguardFilters/issues/191068#issuecomment-2480464336
+        - ^\|\|b\.karte\.io
+        - ^\|\|static\.karte\.io
```

Comment:
I guess if the problem is widely known, it can be removed, so I agree with that.